scripts/nbconverted/tybalt_1layer_model.py

```diff
@@ -283,3 +289,27 @@ def on_epoch_end(self, epoch, logs={}):
 # Save latent space representation
 encoded_rnaseq_df.to_csv(encoded_file, sep='\t')
 
+# Save models
+# (source) https://machinelearningmastery.com/save-load-keras-deep-learning-models/
+
+# Save encoder model
+model_encoder_json = encoder.to_json()
+with open(model_encoder_file, "w") as json_file:
+    json_file.write(model_encoder_json)
+    
```

Comment:
Up to you, I think maybe can also save to hdf5 file
encoder_model_file = path_and_filename
encoder.save(encoder_model_file)

Reply:
I was just following the example for this.  I am not sure what the difference is in the file types...do you have a suggestion?

Reply:
I know hdf5 format is used to save large data (genomic data).. .jason is a format that can be used to store different structures. I think hdf5 maybe more suitable for storing large data, but I am that sure. It is up to you.